corehq/apps/reports/views.py

```diff
@@ -1477,9 +1477,9 @@ def clear_report_caches(request, domain):
 def export_report(request, domain, export_hash, format):
     cache = get_redis_client()
 
-    if cache.exists(export_hash):
```

Comment:
yeah, doesn't work for me locally (have updated all the packages):
`
>>> from dimagi.utils.couch.cache.cache_core import get_redis_client
>>> cache = get_redis_client()
>>> cache.exists('abc')
Traceback (most recent call last):
  File "<console>", line 1, in <module>
AttributeError: 'RedisCache' object has no attribute 'exists'
`